code/game/objects/gamemode/crash/items.dm

```diff
@@ -0,0 +1,206 @@
+/obj/machinery/nuclearbomb/crash
```

Comment:
This in general has a ton of code duplication from the previous one, think it would be possible to reduce some?

Reply:
I agree, I could always remove the old nuke because it wasn't used and just have the crash nuke?

Reply:
Seems good